app/models/analysis.js

```diff
@@ -3,6 +3,10 @@ import { isEmpty } from '@ember/utils';
 import Model, { attr, hasMany, belongsTo }  from '@ember-data/model';
 import ENUMS from 'irene/enums';
 import { t } from 'ember-intl';
+import Inflector from  'ember-inflector';
+
+const inflector = Inflector.inflector;
+inflector.irregular('asvs', 'asvses');
```

Comment:
@guganabu @cosmosgenius need to know one thing, I have written the irregular inflector not in app wide but places where it is required. Is this a better approach or do you want me to shift it to `app.js` ?

Reply:
I think, such conversions are done at per model level is better than having it in global file.